app/assets/javascripts/govuk_publishing_components/lib/cookie-settings.js

```diff
@@ -57,13 +66,15 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       }
     }
 
-    window.GOVUK.setConsentCookie(options)
-    window.GOVUK.setCookie('cookies_preferences_set', true, { days: 365 })
+    if (this.$module.useConsentApi) {
+      window.GOVUK.singleConsent.setPreferences(null, options)
```

Comment:
Unfortunately this single call to `setPreferences` with only the second parameter is outnumbered by the two calls to it in `cookie-banner.js` with only the first parameter. I think if it were switched around it would be more confusing.

Reply:
No worries - I guess maybe the paramaters could be an object instead, but appreciate that I'm probably just introducing more complexity for something that doesn't really matter!